config/internal/api-server/manager/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: manager
+  namespace: system
+  labels:
+    control-plane: api-server
+spec:
+  type: "ClusterIP"
```

Comment:
You don't need to specify this since `ClusterIP` is the default value for Service